CMakeLists.txt

```diff
@@ -533,6 +533,19 @@ if(NRN_ENABLE_NMODL OR NRN_ENABLE_CORENEURON)
 
 endif()
 
+# =============================================================================
+# Flags for using NOCMODL/NMODL
+# =============================================================================
+if(NRN_ENABLE_NMODL)
+  set(NRN_NMODL_--neuron "--neuron")
```

Comment:
This could be `NRN_CODEGENERATOR_FLAGS`, name as is looks a wee bit weird to me. But not a blocker.

Reply:
I agree, it's very clumsy. The reason is that we need the generic FLAGS to refer to all the other flags that apply to both NEURON and CoreNEURON.